Lib/opcode.py

```diff
@@ -291,6 +289,7 @@ def pseudo_op(name, op, real_ops):
 pseudo_op('LOAD_SUPER_METHOD', 263, ['LOAD_SUPER_ATTR'])
 pseudo_op('LOAD_ZERO_SUPER_METHOD', 264, ['LOAD_SUPER_ATTR'])
 pseudo_op('LOAD_ZERO_SUPER_ATTR', 265, ['LOAD_SUPER_ATTR'])
+pseudo_op('LOAD_CLOSURE', 267, ['LOAD_FAST'])
```

Comment:
Let's keep them in increasing numerical order (I'm pretty sure all the other opcodes are).